cypress.config.ts

```diff
@@ -34,8 +34,8 @@ export default defineConfig({
         ...prisonerSearchApi,
         ...curiousApi,
         ...ciagApi,
-        ...educationAndWorkPlanApi,
```

Comment:
`educationAndWorkPlanApi` stub is now unnecessary as this contained stubs for the goal and action plan based APIs.
The induction APIs (that were originally in the CIAG API) are still defined in `ciagApi` stubs. Clearly this is wrong as these APIs now live in PLP / Education & Work Plan; but perhaps we'll rename that file another day 👍 

`learningPlanUi` stub has been removed - it contained a simple/basic stub for any PLP page. I've moved it into a new file `plpUi` which contains stubs for specific PLP UI pages that the integration tests might end up redirecting the user to 👍